src/layout/FileUpload/DropZone/DropzoneComponent.tsx

```diff
@@ -70,9 +70,7 @@ export function DropzoneComponent({
         className={classes.fileUploadTextBoldSmall}
         id={maxSizeLabelId}
       >
-        {`${langAsString('form_filler.file_uploader_max_size')} ${maxFileSizeInMB} ${langAsString(
-          'form_filler.file_uploader_mb',
-        )}`}
+        {`${langAsString('form_filler.file_uploader_max_size')} ${maxFileSizeInMB} MB`}
```

Comment:
I am guessing this is not related and you just removed an unecessary text resource for `MB`. But since you already touched it, I would prefer if the language resource used a parameter for the `maxFileSizeInMB` instead of this contatentation. My suggestion:

1. Change language resource to include a parameter and end with `MB` like this: `file_uploader_max_size: 'Maximum file size {0} MB',`

2. And then use it like this instead:
```suggestion
        <Lang id='form_filler.file_uploader_max_size' params={[maxFileSizeInMB]} />
```